src/popup/components/Modals/ConfirmTransactionSign.vue

```diff
@@ -90,6 +90,39 @@
           />
         </DetailsItem>
 
+        <DetailsItem
+          v-if="popupProps?.tx?.gasLimit"
+          :value="popupProps.tx.gasLimit"
+          :label="$t('transaction.gasLimit')"
+          data-cy="gas"
+        />
+        <DetailsItem
+          v-if="gasPrice"
```

Comment:
`gasPrice` could also exist on Ethereum transactions
If we want those `<DetailsItem>`'s only on for aeternity transactions I suggest to wrap them in a bigger `v-if` and check for protocol. But I believe it would be best to generalize this for all protocols

Reply:
Currently gas price calculation is happening only for the aeternity.